pystarport/pystarport/convert.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+OUTPUT=./proto_python
+COSMOS=./cosmos-sdk
+TENDERMINT=./tendermint
+TMP=$(whereis grpc_python_plugin)
+PLUGIN="$(cut -d' ' -f2 <<<"$TMP")"
+mkdir $OUTPUT
+git clone https://github.com/cosmos/cosmos-sdk.git
+git clone https://github.com/tendermint/tendermint.git
```

Comment:
not sure whether to pin this against master (which can be non-reproducible, but can possibly find out issues earlier) or against specific commits/tags (e.g. v0.40-rc2 for SDK) to match our code

Reply:
`git checkout <tag>`

Reply:
ok, i'll pin the version

Reply:
or `git clone -b v0.40-rc2  https://github.com/cosmos/cosmos-sdk.git` && `git clone -b v0.34-rc5  https://github.com/tendermint/tendermint.git`

Reply:
ok, looking good

Reply:
or `curl ...archive/hash.tar.gz`